start_docker.bat

```diff
@@ -0,0 +1,23 @@
+@echo off
+
+echo.
+echo Launching Docker container
+echo ==========================
+docker run -it --rm^
+ --mount type=bind,source="%cd%"\figures,target=/app/figures^
+ --mount type=bind,source="%cd%"\data,target=/app/data,readonly^
+ --mount type=bind,source="%cd%"\forecasts,target=/app/forecasts,readonly^
+ pycsep_esrl bash
+
+REM Alternative #1:
+REM docker run -it --rm^
+REM  -v "%cd%"\figures:/app/figures -v "%cd%"\data:/app/data -v "%cd%"\forecasts:/app/forecasts^
+REM  pycsep_esrl bash
+
+REM Alternative #2 (passes command to WSL)
+REM wsl docker run -it --rm^
+REM  --mount type=bind,source="$(pwd)"/figures,target=/app/figures^
+REM  --mount type=bind,source="$(pwd)"/results,target=/app/results^
```

Comment:
added this line to be consistent with new changes

Reply:
@wsavran Oh, you would need to add this to the first command in the script, because these latter two are only commented ('REM(ark)') alternatives. Could you take care of that to spare another PR?:
```
 --mount type=bind,source="%cd%"\results,target=/app/results,readonly^
```
Maybe also adjust the first alternative:
```
REM  -v "%cd%"\figures:/app/figures -v "%cd%"\results:/app/results -v "%cd%"\data:/app/data -v "%cd%"\forecasts:/app/forecasts^
```